include/util/static_rtree.hpp

```diff
@@ -43,16 +43,19 @@ namespace util
 template <class EdgeDataT,
           class CoordinateListT = std::vector<Coordinate>,
           bool UseSharedMemory = false,
-          std::uint32_t BRANCHING_FACTOR = 128,
-          std::uint32_t LEAF_NODE_SIZE = 128,
-          std::uint32_t MAX_NUM_CANDIDATES = LEAF_NODE_SIZE * 10>
+          std::uint32_t BRANCHING_FACTOR = 64,
```

Comment:
Hi, here it is a pure rebase issue, i will keep 128. For better performance, it can be run some benchmarks for the full planet R-tree in 2D-space  [BRANCHING_FACTOR, LEAF_NODE_SIZE] and pick the "optimal" value.